CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Changelog
 
+### 4.0.2 (2021-07-21)
+- Improvements:
+	- Avoid displaying campaign message when image can't be downloaded [SDKCF-3977]
+	- Avoid displaying campaign message when display permission request was not successful [SDKCF-3976]
+	- Increase the clickable area of the IAM close button "x" [SDKCF-3958]
```

Comment:
We use past tense for changelogs (see the others below), like "Increased the...".

Reply:
Oops, I knew that 😵